collectors/cgroups.plugin/cgroup-network-helper.sh

```diff
@@ -195,8 +195,9 @@ netnsid_find_all_interfaces_for_pid() {
     local pid="${1}"
     [ -z "${pid}" ] && return 1
 
-    local nsid=$(lsns -t net -p ${pid} -o NETNSID -nr)
-    [ -z "${nsid}" -o "${nsid}" = "unassigned" ] && return 1
+    # shellcheck disable=SC2155
+    local nsid=$(lsns -t net -p "${pid}" -o NETNSID -nr)
+    if [ -z "${nsid}" ] || [ "${nsid}" = "unassigned" ] ; then return 1 ; fi
```

Comment:
Can we omit the space preceding ; as we do in other places?

```suggestion
    if [ -z "${nsid}" ] || [ "${nsid}" = "unassigned" ]; then return 1; fi
```